cacheops/query.py

```diff
@@ -4,6 +4,8 @@
 import threading
 import six
 from random import random
+
+from django.db.models.manager import BaseManager
```

Comment:
Please place this right after `from django.db.models import Model` import.

Reply:
Done